rules/use-iapimarker-with-webapplicationfactory/rule.md

```diff
@@ -30,10 +35,13 @@ This small change leads to a long road of pain:
 2. Which means you need to make your tests internal and
 3. In turn add an `InternalsVisibleTo` tag to your test project for the test runner to be able to access the tests.
 
-#### Option 2 - public partial program class
+### Option 2 - public partial program class
 
 ![](Using-public-partial-program-class.jpg)
```

Comment:
Hey @DevStarOps 

the correct syntax for images is:
```
::: bad
![Figure: Bad example - Using a public partial program class](Using-public-partial-program-class.jpg)
:::

Please fix this one and others ;)
```

Reply:
fixed 😊